### DISCUSSION TITLE
How to "Remove unused JavaScript" with nuxt 3 ?

### DISCUSSION THREAD

**Discussion by johan-Rm** [nuxt/nuxt#16108] [Questions]
I just uploaded a site on Nuxt 3 but the scores with PageSpeed ​​Insights aren't great.

How to "Remove unused JavaScript" with nuxt 3?

**Comment by TheAlexLichter** [comment]
That depends a lot on your website 🙈

**Comment by johan-Rm** [comment]
Yes I understand I will see what I can do to manually remove unused code.

Could this option make things better?

```
  nitro: {
    compressPublicAssets: true
  },
```

**Reply by TheAlexLichter** [reply]
No, this will only compress assets with GZIP/Brotli and not remove content ☺️

**Comment by ambujsahu81** [comment]
UglifyJS and Minification tools such as uglifyjs, Terser can help. [UglifyJS](https://github.com/mishoo/UglifyJS)

This article might be able to help you :- https://www.dawntraoz.com/blog/my-web-performance-journey-with-nuxt-storyblok-netlify/
